src/viur/toolkit/importer/importer.py

```diff
@@ -521,15 +521,15 @@ def set_value(val: t.Any, lang: str | None) -> None:
                     elif bone_value[lang] != val:
                         changes += 1
 
-                    if bone.multiple:
+                    if not bone.multiple:
```

Comment:
I don't get your change. A not multiple bone should not get a `list` as value.


```suggestion
                    if bone.multiple:
```

Reply:
The logic here is, that the algorithm always works with lists. I had the case that the value "True" for a BooleanBone was interpreted as ["T", "r", "u", "e"] some lines below. This was a general bug.

Reply:
Sorry, I cannot follow you.

`["T", "r", "u", "e"]` would be the result of list("True"). But inside of this if, we ensure only an empty ist or put the single-value as is (whether the literal `True` or string `"True"`) inside a list: `list[val]`. This cannot happen here. Are you sure your data is correct?

Reply:
I think that https://github.com/viur-framework/viur-toolkit/pull/41 is a better alternative for this pull request, as the single-value case was not correctly respected in the original sources.

The problem with the `["T", "r", "u", "e"]` as mentioned above resulted within the case 

```py
if isinstance(bone, bones.BooleanBone):
    val = [utils.parse.bool(value) for value in val]
```

which was executed, regardless if val is a list or a single value (like string "True"). 

This was now fixed in the alternative PR.